src/gd.h

```diff
@@ -627,6 +667,14 @@ BGD_DECLARE(gdImagePtr) gdImageCreateFromWebp (FILE * inFile);
 BGD_DECLARE(gdImagePtr) gdImageCreateFromWebpPtr (int size, void *data);
 BGD_DECLARE(gdImagePtr) gdImageCreateFromWebpCtx (gdIOCtx * infile);
 
+BGD_DECLARE(gdImagePtr) gdImageCreateFromHeif (FILE * inFile);
```

Comment:
i know our existing code is inconsistent, but we should start writing new stuff correctly.  so omit the space before the `(` with function calls and after the `*` with pointers.

it comes up in this header and in the gd_heif.c file a bit.

Reply:
Done and thanks for pointing that out.